pg_shard.c

```diff
@@ -937,6 +973,12 @@ BuildDistributedPlan(Query *query, List *shardIntervalList)
 
 		deparse_shard_query(query, shardId, queryString);
 
+		if (LogDistributedStatements)
+		{
+			ereport(LOG, (errmsg("distributed statement: %s", queryString->data),
+						  errhidestmt(true)));
```

Comment:
I _think_ this is so disparate callers can ensure a given _STATEMENT:_ line is only emitted once per stack level. I've looked into my use of it here and it serves no purpose. I'm not suppressing the _STATEMENT:_ line, nor am I trying to print more than one line per shard, so it's not suppressing anything ever. I'll remove it.